spec/controllers/sub_sections_controller_spec.rb

```diff
@@ -31,7 +32,7 @@
       stub_request(:get, raw_content_url)
         .to_return(body: raw_content)
       stub_coronavirus_publishing_api
-      stub_youtube
```

Comment:
Yes it's used in the feature tests. Because I removed the code that tested the live stream object being initialised (which is when the youtube url is validated), I thought I'd leave the database empty for the feature tests so that bit of the code was still being used.